ui/packages/components/tailwind.config.ts

```diff
@@ -15,10 +15,56 @@ export default {
           910: '#0C1323',
           940: '#080D19',
         },
+        primary: {
+          '2xSubtle': 'rgb(var(--color-primary-2xSubtle) / <alpha-value>)',
+          xSubtle: 'rgb(var(--color-primary-xSubtle) / <alpha-value>)',
+          subtle: 'rgb(var(--color-primary-subtle) / <alpha-value>)',
+          moderate: 'rgb(var(--color-primary-moderate) / <alpha-value>)',
+          intense: 'rgb(var(--color-primary-intense) / <alpha-value>)',
+          xIntense: 'rgb(var(--color-primary-xIntense) / <alpha-value>)',
+          '2xIntense': 'rgb(var(--color-primary-2xIntense) / <alpha-value>)',
+        },
+        tertiary: {
+          '2xSubtle': 'rgb(var(--color-tertiary-2xSubtle) / <alpha-value>)',
+          xSubtle: 'rgb(var(--color-tertiary-xSubtle) / <alpha-value>)',
+          subtle: 'rgb(var(--color-tertiary-subtle) / <alpha-value>)',
+          moderate: 'rgb(var(--color-tertiary-moderate) / <alpha-value>)',
+          intense: 'rgb(var(--color-tertiary-intense) / <alpha-value>)',
+          xIntense: 'rgb(var(--color-tertiary-xIntense) / <alpha-value>)',
+          '2xIntense': 'rgb(var(--color-tertiary-2xIntense) / <alpha-value>)',
+        },
+        foreground: {
```

Comment:
I was thinking we could go with the foreground colors as textColor for now and change that that to colors if we really need it in the future. What do you think?

Reply:
I think textColor takes care of most cases, but I needed to use foreground tokens on icon fills...so I added them like this so I could do: `fill-foreground-onContrast` etc. I don't *think* textColor would work in the case...but maybe!

Reply:
Then we could probably do this with [fill](https://tailwindcss.com/docs/theme#configuration-reference), the same way we do it with textColor!